src/GenerateProgrammaticPipelineFromConfig/PipelineFromConfigCommand.php

```diff
@@ -64,16 +61,9 @@ protected function execute(InputInterface $input, OutputInterface $output)
             '<info>Generating programmatic pipeline for an existing Expressive application...</info>'
         );
 
-        try {
-            $generator = new Generator($output);
-            $generator->projectDir = $this->projectDir;
-            $generator->process($this->locateConfigFile($input));
-        } catch (GeneratorException $e) {
-            $console = $this->getErrorConsole($output);
-            $console->writeln('<error>Error during generation:</error>');
-            $console->writeln(sprintf('  <error>%s</error>', $e->getMessage()));
-            return 1;
-        }
+        $generator = new Generator($output);
+        $generator->projectDir = $this->projectDir;
```

Comment:
Actually... no longer necessary. It was necessary before I started mocking the various internal classes, but now it isn't. I'll nuke it in the various `*Command` classes.

It's still necessary for the internal classes that do filesystem operations, though I _may_ be able to move that to constructor arguments. I'll review.